### PR TITLE
MRG: Capture a bunch of warnings

### PR DESCRIPTION
**Pull request by larsoner**
Looking through our Travis logs it's annoying to scroll back through pages of warnings. This should get rid of them at least on the modern `environment.yml` builds.

### CODE REVIEW

File: mne/viz/evoked.py [hunk 1]
```diff
@@ -1310,7 +1310,7 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
 
     # butterfly/time series plot
     # most of this code is about passing defaults on demand
-    ts_ax = fig.add_subplot(212)
+    ts_ax = fig.add_subplot(212, label='timeaxes')
```
Comment (larsoner) [inline mne/viz/evoked.py]: @jona-sassenhagen this is to take care of this warning:
```
mne/viz/tests/test_topo.py::test_plot_topo
  /home/larsoner/custombuilds/matplotlib/lib/matplotlib/cbook/deprecation.py:107: MatplotlibDeprecationWarning: Adding an axes using the same arguments as a previous axes currently reuses the earlier instance.  In a future version, a new instance will always be created and returned.  Meanwhile, this warning can be suppressed, and the future behavior ensured, by passing a unique label to each axes instance.
    warnings.warn(message, mplDeprecation, stacklevel=1)
```
Does it make sense to you?

Comment (jona-sassenhagen) [reply mne/viz/evoked.py]: Actually .. why is that line there at all? The tests I tried pass when it's deleted, plots look fine too, and it seems to be overwriting the identical axes coming from `_prepare_joint_axes` above.

**Comment by larsoner** [comment]
Feel free to push a commit to delete, otherwise I'll do it in a few hours

>

**Comment by agramfort** [comment]
thx @larsoner

**Pull request merged**